src/System.Management.Automation/engine/DataStoreAdapterProvider.cs

```diff
@@ -405,11 +405,7 @@ protected ProviderInfo(ProviderInfo providerInfo)
                 throw PSTraceSource.NewArgumentException("name");
             }
 
-
-            if (String.IsNullOrEmpty(name))
-            {
-                throw PSTraceSource.NewArgumentException("name");
-            }
+            // FIXME: Other checks?
```

Comment:
Please remove this comment and verify that helpfile, description, psSnapIn parameters can be null.

Reply:
Sorry, I think I didn't catch whether I should ensure the `helpfile`, `description` and `psSnapIn` is not null or just remove the comment.

Reply:
@Himura2la Can you either remove this comment or file an issue and update the comment to reference the issue?